gnat2goto/driver/arrays.adb

```diff
@@ -997,4 +997,41 @@ package body Arrays is
                                  Get_Name (Data_Member));
    end Get_Data_Member;
 
+   function Build_Array_Size (Array_Comp : Irep; Idx_Type : Irep) return Irep
+   is (Build_Array_Size (First      => Get_First_Index (Array_Comp),
+                         Last       => Get_Last_Index  (Array_Comp),
+                         Idx_Type   => Idx_Type));
```

Comment:
When you're both moving and changing large blocks of code it does usually make sense to do both in separate commits so it's easier to see what you've changed; In this case it's just between two files so it's not too bad